### DISCUSSION TITLE
How to disable clicking in a ui.table

### DISCUSSION THREAD

**Discussion by vopani** [h2oai/wave#825] [Q&A]
Is there a way to completely disable clicking in a `ui.table`?

I want to display a table with sort, search, filter capabilities but I don't want it to have click triggers at all. I tried setting all `table_column` with `link=False` but double clicking a row still triggers a click.

Any way to configure this?

**Comment by mturoci** [comment]
Hi @vopani, this is currently not possible. Can you provide more context on why would you want it?

**Reply by vopani** [reply]
Because when user clicks (even by mistake), the app will hang. I can always code in some dummy update, but disabling click is a pretty reasonable feature request.

**Reply by mturoci** [reply]
I see, we could add a new prop `clickable_rows` defaulting to `True` which would allow ignoring all the row clicks. Wdyt @lo5 ?

**Reply by mtanco** [reply]
Would this make sense to do a the `ui.table_row` level instead, where each row is clickable or not? 

There are many use cases where you would want to search, sort, and otherwise interact with a table but never need to interact with the server (that is send a row id back to the app). There are a few use cases where you might want a user to click on some rows but not others (for example when manually reviewing data, maybe we would only want the not-yet-reviewed rows to be edited but all rows to be visible)
